docs/framework/data/adonet/dataset-datatable-dataview/security-guidance.md

```diff
@@ -482,3 +482,26 @@ Consider replacing the object model to use [Entity Framework](/ef). Entity Frame
 * Offers built-in protections when deserializing data from untrusted sources.
 
 For apps that use `.aspx` SOAP endpoints, consider changing those endpoints to use [WCF](/dotnet/framework/wcf/). WCF is a more fully featured replacement for `.asmx` web services. WCF endpoints [can be exposed via SOAP](../../../wcf/feature-details/how-to-expose-a-contract-to-soap-and-web-clients.md) for compatibility with existing callers.
+
+## Roslyn static code analyzers
+
+Roslyn analyzer security rules, which run as part of compiling source code, can help find vulnerabilities in C# and Visual Basic related to this security issue. Microsoft.CodeAnalysis.FxCopAnalyzers is a NuGet package of Roslyn analyzers, distributed on [nuget.org](https://www.nuget.org/).
+
+For an overview of Roslyn analyzers, see [Overview of source code analyzers](https://docs.microsoft.com/visualstudio/code-quality/roslyn-analyzers-overview).
+
+The new Microsoft.CodeAnalysis.FxCopAnalyzers rules to enable are:
+- [CA2350](https://docs.microsoft.com/visualstudio/code-quality/ca2350): Do not use DataTable.ReadXml() with untrusted data
+- [CA2351](https://docs.microsoft.com/visualstudio/code-quality/ca2351): Do not use DataSet.ReadXml() with untrusted data
+- [CA2352](https://docs.microsoft.com/visualstudio/code-quality/ca2352): Unsafe DataSet or DataTable in serializable type can be vulnerable to remote code execution attacks
+- [CA2353](https://docs.microsoft.com/visualstudio/code-quality/ca2353): Unsafe DataSet or DataTable in serializable type
+- [CA2354](https://docs.microsoft.com/visualstudio/code-quality/ca2354): Unsafe DataSet or DataTable in deserialized object graph can be vulnerable to remote code execution attacks
+- [CA2355](https://docs.microsoft.com/visualstudio/code-quality/ca2355): Unsafe DataSet or DataTable type found in deserializable object graph
+- [CA2356](https://docs.microsoft.com/visualstudio/code-quality/ca2356): Unsafe DataSet or DataTable type in web deserializable object graph
+- [CA2361](https://docs.microsoft.com/visualstudio/code-quality/ca2361): Ensure autogenerated class containing DataSet.ReadXml() is not used with untrusted data
+- [CA2362](https://docs.microsoft.com/visualstudio/code-quality/ca2362): Unsafe DataSet or DataTable in autogenerated serializable type can be vulnerable to remote code execution attacks
+
+For more information on configuring rules, see [Use rule sets to group code analysis rules](https://docs.microsoft.com/visualstudio/code-quality/using-rule-sets-to-group-code-analysis-rules).
```

Comment:
Rule sets are deprecated in favor of EditorConfig files.

```suggestion
For more information about configuring rules, see [Use Use code analyzers](https://docs.microsoft.com/visualstudio/code-quality/use-roslyn-analyzers).
```

Reply:
Suggestion had a typo, fixed in the latest PR iteration.